AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts

```diff
@@ -365,11 +365,13 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
 
     onDeletionAnswer($event: boolean) {
         this.displayConfirmPopup = false;
+
         if ($event) {
             this.bhClient.deleteNonAvailabilityWorkHours(this.slotToDelete.id).subscribe({
                 next: () => {
                     this.logger.info(`${this.loggerPrefix} Non Working hours deleted`);
                     this.removeSlot();
+                    this.showMessage('Non working hours deleted sucessfully');
```

Comment:
I would suggest naming this `Non-availability hours`, so that it's consistent with the error message below

also you have a typo - `successfully`

Reply:
can we update the tests to assert that the message is shown when the hours are deleted

Reply:
> I would suggest naming this `Non-availability hours`, so that it's consistent with the error message below
> 
> also you have a typo - `successfully`

this is resolved .

Reply:
the typo is still there

Reply:
Added the unit test

Reply:
resolved unit test and typo.